automatoes/acme.py

```diff
@@ -237,7 +237,7 @@ def path(self, path):
 
 class AcmeV2(Acme):
 
-    def __init__(self, url, account, directory="directory", verify=None,
+    def __init__(self, url, account, directory="acme/directory", verify=None,
```

Comment:
That will break compatibility the issue is on the url handling.

Reply:
I know, this is **not** a PR.

Reply:
How about adding a `--directory` option instead of `--server`?

Reply:
I do know and I got it. They are missing several keys on the responses, so they don't follow the rfc.

I'll trust the absolute urls and fix the relative ones, that's the key to fix. 

I've passed the url issue. I'll tweak other issues tonight and ship with 0.9.6 as I need to test if I don't break let's encrypt.